java/flight/pom.xml

```diff
@@ -25,25 +25,17 @@
   </properties>
 
   <dependencies>
-    <dependency>
-      <groupId>com.google.flatbuffers</groupId>
-      <artifactId>flatbuffers-java</artifactId>
-    </dependency>
     <dependency>
       <groupId>org.apache.arrow</groupId>
       <artifactId>arrow-vector</artifactId>
       <version>${project.version}</version>
+      <classifier>${arrow.vector.classifier}</classifier>
```

Comment:
Is this necessary? The shade-plugin can already add a classifier name

Reply:
I'm not sure I understand the question.  This is intended to allow building/running tests with either the shaded or non-shaded artifact.

Reply:
I misunderstood the purpose of this, seems fine